src/printer/index.ts

```diff
@@ -304,7 +305,10 @@ function splitTextToDocs(node: TextNode): Doc[] {
 
 	const textLines = text.split(/[\t\n\f\r ]+/);
 
-	let docs = join(line, textLines).parts.filter((s) => s !== '');
+	// https://github.com/prettier/prettier/blob/33e621133c48e857244edea7f26cf20793c55aa6/src/document/builders.js#L216
+	// in the jsdoc it returns a Doc, but in the code it looks like it returns a Doc[]
```

Comment:
I'm not familiar with types, shouldn't array satisfy `Doc`? https://github.com/prettier/prettier/blob/33e621133c48e857244edea7f26cf20793c55aa6/src/document/builders.js#L27